react/DeliveryPackagesUtils.js

```diff
@@ -38,9 +39,17 @@ function getPackagesLength(chosenPackage) {
   const packages = uniq(
     flatten(
       chosenPackage
-        .filter(li => isDelivery(li) && (!!li.selectedSla || hasSLAs(li)))
-        .map(li => {
-          const sla = li.slas.find(sla => sla.id === li.selectedSla)
+        .filter(
+          (li) =>
+            isDelivery(li) &&
+            (!!li.selectedSla || (hasSLAs(li) && li.slas.some(isDelivery)))
+        )
```

Comment:
nit: I feel like this could be more clear (eg: having a named function just for the `hasSLAs(li) && li.slas.some(isDelivery)` part), but I have no suggestion in mind.

Reply:
maybe `hasDeliverySLAs` 😬